### PR TITLE
QEMU: Include ppc targets (user mode and system mode)

### PR DESCRIPTION
**Pull request by WMCB-Tech**

### CODE REVIEW

File: packages/qemu-system-x86-64-headless/build.sh [hunk 1]
```diff
@@ -33,6 +35,8 @@ termux_step_configure() {
 	QEMU_TARGETS+="aarch64-linux-user,"
 	QEMU_TARGETS+="arm-linux-user,"
 	QEMU_TARGETS+="i386-linux-user,"
+	QEMU_TARGETS+="ppc64-linux-user"
+	QEMU_TARGETS+="ppc-linux-user"
```
Comment (xeffyr) [inline packages/qemu-system-x86-64-headless/build.sh:39]: You have fixed the missing comma for system targets, but what's for user?

**Review by xeffyr — CHANGES_REQUESTED** [review]

File: packages/qemu-system-x86-64-headless/build.sh (hunk 1 above)
Comment (WMCB-Tech) [reply packages/qemu-system-x86-64-headless/build.sh]: Fixed

**Review by xeffyr — CHANGES_REQUESTED** [review]
Everything seems ok now, but still incomplete. See https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#rebuilding-package-with-no-version-change

**Comment by WMCB-Tech** [comment]
@xeffyr done. bumped revision

**Review by xeffyr — APPROVED** [review]

**Pull request merged**